src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.Unit.Tests/Microsoft.Azure.Services.AppAuthentication.Unit.Tests.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
+	<PropertyGroup>
+    
+		<TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    
+		<SignAssembly>True</SignAssembly>
+    
+		<AssemblyOriginatorKeyFile>$(LibraryToolsFolder)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    
+		<DelaySign>True</DelaySign>
+  
+	</PropertyGroup>
+
+
+
+	<ItemGroup>
+    
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    
+		<PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    
+		<PackageReference Include="xunit" Version="2.2.0" />
```

Comment:
@varunsh-msft please model your test project from one of the eixsitng test project and remove redundant properties